hubspot3/tickets.py

```diff
@@ -15,6 +18,15 @@ class TicketsClient(BaseClient):
     :see: https://developers.hubspot.com/docs/methods/tickets/tickets-overview
     """
 
+    __default_batch_properties = ["subject"]
```

Comment:
Are you sure you want to define a name-mangled member? If you want to make it "private" use a single underscore.